portfolio/src/main/webapp/style.css

```diff
@@ -242,6 +242,21 @@ h1 {
   margin-left: 150px;
 }
 
+/* comment section */
+
+.project-name {
+  color: #494949;
+  font-size: 1.3rem;    
```

Comment:
use the font shorthand property where possible: 
https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties

Reply:
I think font-style and font-weight can also be set in font: https://www.w3schools.com/cssref/pr_font_font.asp